LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012, Caktus Consulting Group, LLC
+Copyright (c) 2015, Caktus Consulting Group, LLC
```

Comment:
+1 (though I will have to find something else to do on New Year's Eve now, rather than updating all my copyrights) :)